### DISCUSSION TITLE
Load pages without refresh like facebook

### DISCUSSION THREAD

**Discussion by Neography7** [livewire/livewire#4561] [1. Help]
I want to develop a website but I can't be sure whether I use livewire or not. I want every page will be loaded without refresh like Facebook on this site. Because I use sockets for messaging and notification I don't want these sockets to bring down every page load.

Previously, I made a website like this with jquery ajax but if it is possible, I want to do it with livewire.

So should I use livewire for developing this website? Can I make a single-page application with livewire?

**Comment by denizozturk** [comment]
You can do it with Livewire. But it might be good to check inertia.js first, for what you said, inertia.js might be a better choice.

**Comment by joshhanley** [comment]
@Neography7 yeah I agree with @denizozturk, Livewire was not built for building apps like Facebook, if you need that, then you should consider using React or Vue.

Hope this helps!
